docs/guides/opentracing-jaeger.md

```diff
@@ -0,0 +1,54 @@
+---
+id: opentracing-jaeger
+title: OpenTracing with Jaeger
+---
+
+Saleor has an implementation of OpenTracing used for tracing performance of the application. Jaeger is the currently implemented provider which allows to recap whats collected through dedicated web interface.
```

Comment:
```suggestion
Saleor has an implementation of OpenTracing used to trace performance of the application. Jaeger is the currently implemented provider which allows to recap what is collected through a dedicated web interface.
```

Reply:
Grammarly gave me this suggestion for the original text:

"Saleor has an implementation of OpenTracing used for tracing the performance of the application. Jaeger is the currently implemented provider that allows recapping what's collected through a dedicated web interface."

Reply:
Sounds good